docs/DeveloperGuide.md

```diff
@@ -1,30 +1,116 @@
 # Developer Guide
 
+---
+Welcome to the PlanPal Developer Guide! Thank you for taking an interest in the behind-the-scenes work of our product. We hope this document proves informative and useful for your work!
+
+## Table of Contents
+{to be inserted at a later date}
+
+---
+
 ## Acknowledgements
 
+
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
```

Comment:
I understand that this is for planning purposes, but this can be removed once information has been added to the DG